daemon/mgr/container_exec.go

```diff
@@ -27,18 +29,48 @@ func (mgr *ContainerManager) CreateExec(ctx context.Context, name string, config
 		return "", fmt.Errorf("container %s is not running", c.ID)
 	}
 
+	Envs := MergeEnvs(c.Config.Env, config.Env)
+	logrus.Info("Envs", Envs)
```

Comment:
This line of log is `Info`, and I think it is too casual. It will add more info log in production. How about changing this into `Debugf`?